hasura.planx.uk/migrations/1707234695689_create_view_feedback_summary/up.sql

```diff
@@ -0,0 +1,46 @@
+CREATE OR REPLACE VIEW "public"."feedback_summary" AS
+SELECT 
+    fb.id AS feedback_id,
+    t.name AS team,
+    f.slug AS service_slug,
+    fb.created_at,
+    fb.node_id,
+    fb.device,
+    fb.user_context,
+    fb.user_comment,
+    fb.feedback_type,
+    fb.status,
+    fb.node_type,
+    COALESCE(
+        published_flow_node.data ->> 'title', 
+        published_flow_node.data ->> 'text', 
+        published_flow_node.data ->> 'flagSet'
+    ) AS node_title,
+    published_flow_node.data ->> 'description' AS node_text,
+    published_flow_node.data ->> 'info' AS help_text,
+    published_flow_node.data ->> 'policyRef' AS help_sources,
+    published_flow_node.data ->> 'howMeasured' AS help_definition,
+    (fb.user_data -> 'passport' -> 'data' -> '_address' ->> 'single_line_address') AS street_address,
+    (fb.user_data -> 'passport' -> 'data' -> '_address' ->> 'uprn') AS UPRN,
```

Comment:
nit: without quotes, postgres evaluates this as lowercase `uprn` - can you update casing here for consistency/readability?

Reply:
I've updated for it to be `uprn` to be consistent with everything else, can I just double check this is what you were looking for rather than making it `"UPRN"`?

Reply:
yes exactly - lowercase is ideal default in my opinion, needing to quote when querying is a pain!